docs/overview.md

```diff
@@ -0,0 +1,160 @@
+# Overview
```

Comment:
Either way is correct but I would prefer to keep the overview in the Telemetry.Metrics module doc. Then everything else (rationale, custom formatters, etc) can be a custom document. The idea is that most users need to read the overview, so I would put it in the place they are likely to see it the most. Thoughts?

Reply:
The overview is the default page, so everyone will stumble upon it when opening the docs. I made it this way, because I got feedback that previous moduledoc was quite long and it was hard to find short information on how to start.

I imagine that the users will see the overview first, but when they already know the lib, they will find essential information in the moduledoc. What do you think?